robottelo/constants/__init__.py

```diff
@@ -1542,7 +1541,7 @@
         'create_hosts',
         'edit_hosts',
         'destroy_hosts',
-        'build_hosts' 'power_hosts',
+        'build_hostspower_hosts',
```

Comment:
According to the list of available permissions, these are two separate permissions
```suggestion
        'build_hosts',
        'power_hosts',
```